include/seqan3/std/span

```diff
@@ -17,16 +17,19 @@
 //!\cond
 #pragma once
 
+#if __has_include(<span>)
```

Comment:
This will likely not be enough, but we can fix it later.

Reply:
agreed